infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/replication/receive/LogReplicationSinkManager.java

```diff
@@ -363,14 +365,13 @@ private boolean receivedValidMessage(LogReplicationEntry message) {
      * 1. update the metadata store with the most recent topologyConfigId
      * 2. reset snapshotWriter and logEntryWriter state
      * 3. reset buffer logEntryBuffer state.
-     * @param active
-     * @param siteConfigID
+     * @param topologyConfigId
      */
-    public void updateTopologyConfigId(boolean active, long siteConfigID) {
-        this.siteConfigID = siteConfigID;
+    public void updateTopologyConfigId(long topologyConfigId) {
```

Comment:
this is the method I mentioned up in another comment that we might change the name as it is overloaded with other functionalities.

Reply:
Moved metadata manager update to discovery service.